bin/main.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../lib/enumerables'
+
+class MyList < MyEnumerable
```

Comment:
- [ ] MyEnumerable module should be included in MyList class, here MyList is inheriting from MyEnumerable, kindly change this.